cmd/insprd/memory/brokers/brokers.go

```diff
@@ -3,39 +3,37 @@ package brokers
 import (
 	"go.uber.org/zap"
 	"inspr.dev/inspr/cmd/sidecars"
+	apimodels "inspr.dev/inspr/pkg/api/models"
 	"inspr.dev/inspr/pkg/ierrors"
 	"inspr.dev/inspr/pkg/meta/brokers"
 	"inspr.dev/inspr/pkg/sidecars/models"
-	"inspr.dev/inspr/pkg/utils"
 )
 
-// GetAll returns an array containing all currently configured brokers
-func (bmm *BrokerMemoryManager) GetAll() (utils.StringArray, error) {
+// Get returns the brokers configured data
+func (bmm *brokerMemoryManager) Get() (*apimodels.BrokersDI, error) {
+	bmm.availible.Lock()
+	defer bmm.availible.Unlock()
```

Comment:
Is there need to lock the mutex for a get command? It doesn't alter the structure in any way